utils/utils.go

```diff
@@ -26,6 +27,9 @@ const (
 	PngMimeType = "image/png"
 
 	TextRegex = `^[a-zA-Z0-9 +.,;:?!'"\-_/()\[\]~&#$—]+$`
+
+	// Limit Http response to 1 MB
+	httpResponseLimit = 1 * 1024 * 1024
```

Comment:
rename to `httpResponseLimitBytes` to make explicit?

Reply:
92cb205981d0e7f29408d4d8fcbb3fd2f7450a10